model/project.go

```
@@ -1450,8 +1476,15 @@ func (p *Project) BuildProjectTVPairs(patchDoc *patch.Patch, alias string) {
 // mapping of the build variant to the tasks that will run on that build
 // variant. If includeDeps is set, it will also resolve task dependencies.
 func (p *Project) ResolvePatchVTs(patchDoc *patch.Patch, requester, alias string, includeDeps bool) (resolvedBVs []string, resolvedTasks []string, vts []patch.VariantTasks) {
-	bvs := patchDoc.BuildVariants
-	tasks := patchDoc.Tasks
+	var bvs, tasks []string
+	for _, bv := range patchDoc.BuildVariants {
+		bvRegex, _ := regexp.Compile(bv)
+		bvs = append(bvs, p.FindMatchingBuildVariants(bvRegex)...)
+	}
+	for _, t := range patchDoc.Tasks {
+		tRegex, _ := regexp.Compile(t)
```

Same thing about not ignoring errors.

---

same as above